audio-router/src/main/java/com/twilio/audio_router/AudioDevice.java

```diff
@@ -0,0 +1,39 @@
+package com.twilio.audio_router;
```

Comment:
Sorry to nitpick but should this be `com.twilio.audiorouter`

This follows the example here "myapp": https://developer.android.com/studio/build/application-id

Reply:
Yeah I was following the java package naming convention defined here: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

I also noticed android studio automatically added an underscore to the package name when I added a hyphen to the module "audio-router". I guess I can just name this thing "switch" already and we wouldn't have to worry about it 😆